### PR TITLE
Update doc for ACR credential helper configuration

### LINKED ISSUE #1530: Doc update for Azure credential helper
Azure credential helper is now supported (#1511).

https://github.com/GoogleContainerTools/jib/pull/1524#issuecomment-469478073

### PR DESCRIPTION
**Pull request by chanseokoh**
Fixes #1530.

@andxu following the same template and pattern for other credential helper documentation, I did not list the detailed steps to configure the ACR credential helper. I think we'll keep this simple and consistent.

### CODE REVIEW

**Comment by chanseokoh** [comment]
@andxu I also think `<credHelper>acr-windows</credHelper>` is unnecessary as long as `acr login` configures `~/.docker/config.json` correctly.

File: jib-gradle-plugin/README.md [hunk 1]
```diff
@@ -99,6 +98,16 @@ For example, to build the image `my-docker-id/my-app`, the configuration would b
 jib.to.image = 'my-docker-id/my-app'
 ```
 
+### Using [Azure Container Registry (ACR)](https://azure.microsoft.com/en-us/services/container-registry/)...
```
Comment (TadCordle) [inline jib-gradle-plugin/README.md]: Header is slightly bigger than the others in this section, use `####`.

File: jib-gradle-plugin/README.md [hunk 2]
```diff
@@ -99,6 +98,16 @@ For example, to build the image `my-docker-id/my-app`, the configuration would b
...
+
+*Make sure you have a [`ACR Docker Credential Helper`](https://github.com/Azure/acr-docker-credential-helper) installed and set up. For example, on Windows, the credential helper would be `docker-credential-acr-windows`. See [Authentication Methods](#authentication-methods) for other ways of authenticating.*
+
+For example, to build the image `my_acr_name.azurecr.io/my-app`, the configuration would be:
+
+```groovy
+jib.to.image = 'my_acr_name.azurecr.io/my-app'
+```
+
 #### *TODO: Add more examples for common registries.*
```
Comment (TadCordle) [inline jib-gradle-plugin/README.md]: Do we need this TODO anymore?

File: jib-maven-plugin/README.md [hunk 3]
```diff
@@ -124,6 +123,20 @@ For example, to build the image `my-docker-id/my-app`, the configuration would b
 </configuration>
 ```
 
+### Using [Azure Container Registry (ACR)](https://azure.microsoft.com/en-us/services/container-registry/)...
```
Comment (TadCordle) [inline jib-maven-plugin/README.md]: Same thing about the header size

File: jib-maven-plugin/README.md [hunk 4]
```diff
@@ -124,6 +123,20 @@ For example, to build the image `my-docker-id/my-app`, the configuration would b
...
+
+```xml
+<configuration>
+  <to>
+    <image>my_acr_name.azurecr.io/my-app</image>
+  </to>
+</configuration>
+```
+
 #### *TODO: Add more examples for common registries.*
```
Comment (TadCordle) [inline jib-maven-plugin/README.md]: Same thing about the TODO

File: jib-gradle-plugin/README.md (hunk 1 above)
Comment (chanseokoh) [reply jib-gradle-plugin/README.md]: Oops, good catch.

File: jib-gradle-plugin/README.md (hunk 2 above)
Comment (chanseokoh) [reply jib-gradle-plugin/README.md]: Removed.

**Comment by andxu** [comment]
@chanseokoh The document looks good to me, thanks

**Comment by andxu** [comment]
@chanseokoh , yes, `<credHelper>acr-windows</credHelper>` is unnecessary, the config.json will set the right credential store for the file in `~/.docker/config.json` during the installation of acr helper

**Pull request merged**